### PR TITLE
Kvg smrtlink updates

### PR DESCRIPTION
**Pull request by kvg**

### CODE REVIEW

File: docker/lr-pb/detect_run_info.py [hunk 1]
```diff
@@ -120,8 +120,15 @@ def main():
                     movieName = info['CollectionMetadata.Context']
                     readType = "SUBREAD"
```
Comment (SHuang-Broad) [inline docker/lr-pb/detect_run_info.py:122]: Let's take away commented-out code

File: docker/lr-pb/Makefile [hunk 2]
```diff
@@ -1,17 +1,17 @@
 IMAGE_NAME = lr-pb
-VERSION = 0.1.37
+VERSION = 0.1.38
```
Comment (SHuang-Broad) [inline docker/lr-pb/Makefile:2]: We are already having a fragmentation problem for this docker.
https://github.com/broadinstitute/long-read-pipelines/search?l=wdl&q=lr-pb

**Review by SHuang-Broad — APPROVED** [review]
one trivial change, one question, and one comment (we need to address this together as a team).

**Review by bshifaw — APPROVED** [review]
looks good

**Pull request merged**